llvm/lib/CodeGen/TargetLoweringObjectFileImpl.cpp

```diff
@@ -2289,25 +2314,6 @@ bool TargetLoweringObjectFileWasm::shouldPutJumpTableInFunctionSection(
   return false;
 }
 
-const MCExpr *TargetLoweringObjectFileWasm::lowerRelativeReference(
```

Comment:
I'll pre-commit this. wasm and xcoff did cargo culting and added the unneeded functions.